contracts/core/LensHub.sol

```diff
@@ -93,6 +93,7 @@ contract LensHub is LensNFTBase, VersionedInitializable, LensMultiState, LensHub
         if (msg.sender == _emergencyAdmin) {
             if (newState == DataTypes.ProtocolState.Unpaused)
                 revert Errors.EmergencyAdminCannotUnpause();
+            _validateNotPaused();
```

Comment:
Worth to simplify the logic to this?
```solidity
function setState(DataTypes.ProtocolState newState) external override {
    if (msg.sender == _emergencyAdmin && newState < _state) {
        revert Errors.EmergencyAdminCannotUnpause();
    } else if (msg.sender != _governance) {
        revert Errors.NotGovernanceOrEmergencyAdmin();
    }
    _setState(newState);
}
```

As our states are defined as enum, we can just simplify the logic to ask `newState > _state` for emergency admin, which basically means it can only "Pause more" but not "Unpause".
```solidity
    enum ProtocolState {
        Unpaused,
        PublishingPaused,
        Paused
    }
```

Reply:
We keep it as it is to keep contract size as low as possible